test/hotspot/jtreg/compiler/unsafe/UnsafeCopyMemory.java

```diff
@@ -28,6 +28,7 @@
  * @modules java.base/jdk.internal.misc
  *
  * @run main/othervm -Xbatch -XX:CompileCommand=dontinline,compiler.unsafe.UnsafeCopyMemory::test*
+ *                   -XX:+UnlockDiagnosticVMOptions -XX:+StressGCM -XX:+StressLCM
```

Comment:
You should add `@key stress` to the test.

Reply:
Fixed.